src/post/post.service.ts

```diff
@@ -26,7 +26,11 @@ export class PostService {
     return `This action updates a #${id} post`;
   }
 
-  remove(id: number) {
-    return `This action removes a #${id} post`;
+  async remove(id: string) {
+    try {
+      await this.postModel.findByIdAndDelete(id);
```

Comment:
why do you use postModel instead of postRepository like user.service.ts?

Reply:
Because Post is a schema in mongoDB and User is a entity in Mysql. Schema use model and entity use repository.

Reply:
@NhatNguyen1502 It seems like we can use typeorm with mongodb. Can you research and try to apply it

Reply:
@vanglevan Is it necessary to change to repository? I see in NestJS's documentation that when using mongoose, they use model instead of repository.

Reply:
@NhatNguyen1502  ahh, I have already thought you use with mongodb native, my mistake, sorry.
If using with mongoose, it's is okay